packages/components/src/Icons/custom/ArrowDownwardRounded.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M11 5V16.17L6.11999 11.29C5.72999 10.9 5.08999 10.9 4.69999 11.29C4.30999 11.68 4.30999 12.31 4.69999 12.7L11.29 19.29C11.68 19.68 12.31 19.68 12.7 19.29L19.29 12.7C19.68 12.31 19.68 11.68 19.29 11.29C18.9 10.9 18.27 10.9 17.88 11.29L13 16.17V5C13 4.45 12.55 4 12 4C11.45 4 11 4.45 11 5Z" />,
```

Comment:
I got this path from the UI kit: https://www.figma.com/file/YPTOEVRuas5ytXQEcwm4FK/UI-Kit-Sticker-Sheet-Core?node-id=649%3A2126

Here's the blocky version in `traject`: https://github.com/FB-PLP/traject/blob/77d303699345622a3f8e0aebe013fbf3c1a7d421/app/assets/javascripts/v2/core/EDSCandidates/SvgIcon/ArrowDownward.jsx

Reply:
ohh shoot, sorry I didn't clarify this more. Material UI published component versions of these icons that we can use -- I added all of the rounded ones in https://github.com/chanzuckerberg/edu-design-system/releases/tag/all-rounded-icons, and then only committed the icons in the UI kit at that time.

For consistency, we should do
```
git fetch --tags
git co tags/all-rounded-icons -- packages/components/src/Icons/<IconName>.tsx
```

Then any icon that wasn't published by MUI should go within this `custom/` folder (I think only `StarCircleRounded` isn't there)